### PR TITLE
Implementation for #7

### PR DESCRIPTION
**Pull request by JahlomP**

### CODE REVIEW

File: plugins/pencilblue/controllers/index.js [hunk 1]
```diff
@@ -275,10 +275,15 @@ Index.prototype.renderContent = function(content, contentSettings, themeSettings
...
+        if (isPage || !contentSettings.allow_comments) {
            cb(null, '');
            return;
-       }
+        }
+
+		if (!content.allow_comments) {
+			cb(null, '');
+			return;
+		}
```
Comment (brianhyder) [inline plugins/pencilblue/controllers/index.js]: This is really nit-picky but as I've come across them I've been trying to do ```return cb(null, '');``` to try and save a line.  Would you mind changing that?

Comment (JahlomP) [inline plugins/pencilblue/controllers/index.js]: ok

**Comment by JahlomP** [comment]
All done

File: plugins/pencilblue/controllers/index.js [hunk 2]
```diff
@@ -275,10 +275,13 @@ Index.prototype.renderContent = function(content, contentSettings, themeSettings
...
-           cb(null, '');
-           return;
-       }
+        if (isPage || !contentSettings.allow_comments) {
+           return (null, '');
+        }
+
+	if (!content.allow_comments) {
+	   return cb(null, '');
+	}
```
Comment (brianhyder) [inline plugins/pencilblue/controllers/index.js]: It looks good to me.  I'll give @blakecallens a chance to comment before I merge.  

One more small change.  Could you please combine the if statements?

```
if (isPage || !contentSettings.allow_comments || !content.allow_comments) {
  return cb(null, '');
}
```

Comment (JahlomP) [inline plugins/pencilblue/controllers/index.js]: Ok

**Comment by JahlomP** [comment]
done

File: public/localization/en-us.js [hunk 3]
```diff
@@ -300,7 +300,8 @@ var loc =
         INACTIVE_SECTIONS: 'Drag unassociated sections here',
         PUBLISHED: 'Published',
         UNPUBLISHED: 'Unpublished',
-        DRAFT: 'Draft'
+        DRAFT: 'Draft',
+        ALLOW_COMMENTS: 'Allow comments?'
```
Comment (blakecallens) [inline public/localization/en-us.js]: The question mark should be removed to be in line with our other localization.

File: plugins/pencilblue/controllers/index.js [hunk 4]
```diff
@@ -275,10 +275,9 @@ Index.prototype.renderContent = function(content, contentSettings, themeSettings
     ats.registerLocal('author_position', content.author_position ? content.author_position : '');
     ats.registerLocal('media_body_style', content.media_body_style ? content.media_body_style : '');
     ats.registerLocal('comments', function(flag, cb) {
-       if (isPage || !contentSettings.allow_comments) {
-           cb(null, '');
-           return;
-       }
+        if (isPage || !contentSettings.allow_comments || !content.allow_comments) {
```
Comment (blakecallens) [inline plugins/pencilblue/controllers/index.js]: If someone upgrades and they have articles saved in the system from before this functionality, they will never show comments. I suggest changing this to:
```JavaScript
if (isPage || !contentSettings.allow_comments || content.allow_comments === false) {
```

**Comment by blakecallens** [comment]
Added a few requests for minor changes.

**Comment by blakecallens** [comment]
Thanks @JahlomP. I'm good with a merge, @brianhyder.

**Pull request merged**